packages/nextjs/app/builders/0x357bc010520575Ea5720aC3F7cb48fcedfc711Ed/page.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import React from "react";
+import { NextPage } from "next";
+import { BriefcaseIcon, CodeBracketIcon } from "@heroicons/react/20/solid";
+
+const handleClick1 = () => {
+  window.open("https://github.com/Selpak99", "_blank", "noopener,noreferrer");
+};
+
+const handleClick2 = () => {
+  window.open(
+    "https://app.buidlguidl.com/builders/0x357bc010520575Ea5720aC3F7cb48fcedfc711Ed",
+    "_blank",
+    "noopener,noreferrer",
+  );
+};
```

Comment:
Hey @Selpak99 - i think we can remove this and use html anchor tag instead of doing it with javascript

You could keep button className as is just change button to a tag 

Checkout 
- https://github.com/BuidlGuidl/batch8.buidlguidl.com/pull/24#discussion_r1726164292
- https://github.com/BuidlGuidl/batch8.buidlguidl.com/pull/24#discussion_r1726166418